packages/dds/matrix/src/test/undoRedoStackManager.ts

```diff
@@ -48,6 +48,15 @@ class Stack<T> {
 			this.itemPushedCallback();
 		}
 	}
+
+	public get length(): number {
```

Comment:
Nit: could you add docs to the new methods? 😁

Reply:
Added